daemon/archive.go

```diff
@@ -297,6 +325,9 @@ func (daemon *Daemon) containerExtractToDir(container *container.Container, path
 }
 
 func (daemon *Daemon) containerCopy(container *container.Container, resource string) (rc io.ReadCloser, err error) {
+	if resource[0] == '/' || resource[0] == '\\' {
+		resource = resource[1:]
+	}
```

Comment:
If I recall, this was to do with the error message being weird. This is pulled from `ContainerCopy` into this `containerCopy` which is where the implementation is.